src/main/java/htsjdk/samtools/BinningIndexBuilder.java

```diff
@@ -33,6 +34,7 @@
  * Builder for a BinningIndexContent object.
  */
 public class BinningIndexBuilder {
+    private static final int UNINITIALIZED_WINDOW = -1;
```

Comment:
Done. There were previously no unit tests so no there are tests for a few basic cases.